consent_manager/js_api/commands.rst

```diff
@@ -0,0 +1,198 @@
+    dataLayer.push({'event': 'ppms.cm:setComplianceSettings', parameters: [settings, onFulfilled, onRejected]});
+
+`dataLayer.push` interface is only for backward compatibility and you can read more about this particular case below. We recommend ``ppms.cm.api``.
+
+.. object:: settings
+
+    **required** The consent settings object.
+
+        Example::
+
+            {consents: {analytics: {status: 1}}}
+
+    Where `consent.analytics` is consent type and status indicate:
+
+    * `0` - user has rejected the consent
+    * `1` - user has approved the consent
+
+.. function:: onFulfilled()
+
+     **required** The fulfilment handler callback. This function is **required**.
+
+.. function:: onRejected(error)
+
+    The rejection handler callback (called with error code). If not specified, exception will be thrown in main stacktrace.
+
+    :param string|object error: **Required** Error code or exception.
+
+Get compliance settings
+```````````````````````
+Return current privacy settings. Might be useful for initializing custom decision view.
+When there is no decisions, just returns empty object. This state can be used to detect first time user interaction with consent mechanism.
+
+Code::
+
+    ppms.cm.api('getComplianceSettings', onFulfilled, onRejected);
+    dataLayer.push({'event': 'ppms.cm:getComplianceSettings', parameters: [onFulfilled, onRejected]});
+
+`dataLayer.push` interface is only for backward compatibility and you can read more about this particular case below. We recommend ``ppms.cm.api``.
+
+.. object:: settings
+
+     **required** The consent settings object.
+
+        Example::
+
+            {consents: {analytics: {status: -1, updatedAt: '2018-07-03T12:18:19.957Z'}}}
+
+    Where `consent.analytics` is consent type and status indicate:
+
+    * `-1` - user has not interacted, e.g. has closed a consent popup without any decision
+    * `0` - user reject consent
+    * `1` - user approve consent
+
+.. function:: onFulfilled(settings)
+
+    **required** The fulfilment handler callback (called with result).
+
+.. function:: onRejected(error)
+
+    The rejection handler callback (called with error code). If not specified, exception will be thrown in main stacktrace.
+
+    :param string|object error: **Required** Error code or exception.
+
+Send data subject request
+`````````````````````````
+Command send data subject request to Consent Manager collector.
+
+Code::
+
+    ppms.cm.api('sendDataRequest', request, onFulfilled, onRejected);
+    dataLayer.push({'event': 'ppms.cm:sendDataRequest', parameters: [request, onFulfilled, onRejected]});
+
+`dataLayer.push` interface is only for backward compatibility and you can read more about this particular case below. We recommend ``ppms.cm.api``.
+
+.. object:: request
+
+    **required** The subject data request.
+
+        Example::
+
+            {content: '', email: '', type: 'change_data|view_data|delete_data'}
```

Comment:
I suspect that value provided in `type` is exclusive (it should be one string from provided options), but it's not clear because example isn't best place to convey that. Maybe you can explain it in the description?
![2019-08-29T155526](https://user-images.githubusercontent.com/4488774/63946460-77b75a00-ca75-11e9-84d7-d7a8cbc0c63f.png)

Reply:
I've improved this part a little,
if you fancy the change we can merge